### DISCUSSION TITLE
code blocks?

### DISCUSSION THREAD

**Discussion by jersam** [karakeep-app/karakeep#480] [Q&A] [answered]
hoping to migrate my saved files in notepad++ to hoarder.

what am I missing to bring over say my docker stack files to hoarder? I drop the text in and it trys to pretty it up. 

possible to enable code blocks for a whole note? or plain text?

**Comment by kamtschatka** [comment, marked as answer]
you can use markdown for code blocks. 
Use ``` for codeblocks
![image](https://github.com/user-attachments/assets/3d6f03ff-0329-4a21-b5f1-6a2cae5ce1f9)

**Comment by jersam** [comment]
I damnit. thank you. I was doing... ''' not ```
